### PR TITLE
fixes for having reasonable path for trycyp loging

### PR DESCRIPTION
**Pull request by zippy**
## PR summary

For gathering log information from many remote servers being managed by trycp, the log files need to be at a known path.  This PR updates it so logs can be pulled from:

`/tmp/trycp/conductors/\.tmp[a-zA-Z0-9]*/[0-9]+/stderr.txt`

## testing/benchmarking notes

( if any manual testing or benchmarking was/should be done, add notes and/or screenshots here )

## followups

( any new tickets/concerns that were discovered or created during this work but aren't in scope for review here )

## changelog

- [ ] if this is a code change that effects some consumer (e.g. zome developers) of holochain core,  then it has been added to [our between-release changelog](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG-UNRELEASED.md) with the format 

```markdown
- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)
```

## documentation

- [ ] this code has been documented according to our [docs checklist](https://hackmd.io/@freesig/Hk9AmKJNS)

### CODE REVIEW

File: crates/trycp_server/src/main.rs [hunk 1]
```diff
@@ -80,11 +82,22 @@ struct TrycpServer {
     port_range: PortRange,
 }
 
+fn make_conductor_dir() -> PathBuf {
+    let conductor_path = PathBuf::new().join(TRYCP_DIRNAME).join(CONDUCTORS_DIRNAME);
+    std::fs::create_dir_all(conductor_path.clone()).expect("can create conductors dir");
```
Comment (struktured) [inline crates/trycp_server/src/main.rs]: This part should probably be a _create if it doesn't exist already_ pattern.

File: crates/trycp_server/src/main.rs [hunk 2]
```diff
@@ -80,11 +82,22 @@ struct TrycpServer {
...
+    let dir = Builder::new().tempdir_in(conductor_path).expect("should create conductor tmp dir").into_path();
```
Comment (struktured) [inline crates/trycp_server/src/main.rs]: This is always random so shouldn't ever fail due to uniqueness issues, unlike the previous step.

File: crates/trycp_server/src/main.rs [hunk 3]
```diff
@@ -80,11 +82,22 @@ struct TrycpServer {
...
+    dir
+}
+
 impl TrycpServer {
     pub fn new(port_range: PortRange) -> Self {
+        let root_path = PathBuf::new().join(TRYCP_DIRNAME);
+        std::fs::create_dir_all(root_path.clone()).expect("can create trycp dir");
+        let dna_dir = root_path.join(DNAS_DIRNAME);
```
Comment (maackle) [inline crates/trycp_server/src/main.rs]: It would feel cozier if there were also a `make_dna_dir()` function alongside `make_conductor_dir()`

Comment (zippy) [reply crates/trycp_server/src/main.rs]: done

File: crates/trycp_server/src/main.rs (hunk 1 above)
Comment (zippy) [reply crates/trycp_server/src/main.rs]: I think that's exactly what create_dir_all does.  If it's already there it won't panic

**Pull request merged**